lib/shared/runtime-info.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * @fileoverview Utility to get information about the execution environment.
+ * @author Kai Cataldo
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const path = require("path");
+const spawn = require("cross-spawn");
+const { isEmpty } = require("lodash");
+const log = require("../shared/logging");
+const packageJson = require("../../package.json");
```

Comment:
Destructuring assignment syntax seems to be more concise IMO :thinking:

Reply:
This could work if we did `const packageJsonVersion = require("../../package.json").version;` or `const { version: packageJsonVersion } = require("../../package.json");`, but I'm not sure those an improvement over the current code!

Reply:
I find it to be a lot concise the original version. IMO the already defined function in scope ( with the same name) could be renamed a bit. Well, let's wait to hear out from other maintainers.
/cc @platinumazure @ilyavolodin

Reply:
I find it to be a lot more concise than the original version. IMO the already defined function in scope ( with the same name) could be renamed a bit. Well, let's wait to hear out from other maintainers.
/cc @platinumazure @ilyavolodin

Reply:
I don't have a strong opinion about whether we use destructuring or not here, but I don't think it's a good idea to change the public API of this module for the sake of making the code more concise (particularly since there won't be any difference in performance).

Reply:
I've always been a fan of explicit over implicit, even at the cost of conciseness. (For that reason, I almost never use destructuring, myself.) So I like the current change here.

Reply:
I think it's a matter of personal preference. Some people prefer destructuring, some prefer direct assignments. I'm in the latter camp, unless you are doing more then one assignment from a single variable. So I prefer currently implemented style. But again, personal opinion only.

Reply:
Since it’s a style issue and it sounds like all the team members are on board with the current version, I’m going to leave it as is. Thanks!